app/assets/stylesheets/bootstrap_and_overrides.scss

```diff
@@ -0,0 +1,24 @@
+@import 'twitter-bootstrap-static/bootstrap';
+@import 'twitter-bootstrap-static/fontawesome';
+
+body {
+  background: #f3f4f5;
+}
+
+// Make navigation & page header background white
+.navbar-default, ._page-header {
```

Comment:
Selector `_page-header` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own line